bubble_sort/Python/bubble_sort.py

```diff
@@ -0,0 +1,14 @@
+def bubbleSort(arr): 
+    n = len(arr) 
+    for i in range(n): 
+        for j in range(0, n-i-1): 
+            if arr[j] > arr[j+1] : 
+                arr[j], arr[j+1] = arr[j+1], arr[j] 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpw3ovi5n_/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpw3ovi5n_/bubble_sort/Python/bubble_sort.py
@@ -3,7 +3,7 @@
     for i in range(n): 
         for j in range(0, n-i-1): 
             if arr[j] > arr[j+1] : 
-                arr[j], arr[j+1] = arr[j+1], arr[j] 
+                arr[j], arr[j+1] = arr[j+1], arr[j]
 
 a = [76, 74, 63, 74, 24, 93, 25]
 
```